srcpkgs/dumb/template

```diff
@@ -42,3 +54,13 @@ dumb-devel_package() {
 		vmove usr/lib/pkgconfig
 	}
 }
+
+if [ "$build_option_examples" ]; then
+	dumb-examples_package() {
+		depends="argtable SDL2"
```

Comment:
I'd say this depends line is still redundant

Reply:
Removed.